tests/system_tests_qdmanage.py

```diff
@@ -515,5 +514,66 @@ def test_create_delete_address_pattern(self):
                 for p in config:
                     self.assertNotEqual(p[0], pattern)
 
+    def test_yy_query_many_links(self):
+        # This test will fail without the fix for DISPATCH-974
+        c = BlockingConnection(self.address())
+        count = 0
+        links = []
+        COUNT = 5000
+
+        print ("Creating 10,000 links, please wait")
```

Comment:
These prints won't necessarily appear on the console since stdout is by default buffered.

You can add a call to sys.stdout.flush() after each print to force the output immediately.

Personally I'm not a big fan of printing debug stuff during the tests - unless you can figure out a way to only do it when ctest's "-V" flag has be specified.

Reply:
Removed all prints, they are GONE!